packages/ffe-icons-react/src/build-tsx-components.js

```diff
@@ -59,19 +59,17 @@ const svg = ${toTsx(icons[iconName])};
 
 interface IconProps extends React.SVGAttributes<SVGElement> {
     desc?: string;
-    focusable? : boolean;
```

Comment:
@steffenz  So, `focusable` is a duplicate because it is inherited from `React.SVGAttributes`?

Reply:
That's right! I probably should have updated the description for that. This doesn't seem to break anything, and the new inherited "focusable" prop will accept both boolean and string enum values (allowing values such as as "true", "false" and "auto"). I didn't remove it from the PropTypes however to make this available to non-typescript users and the styleguidist documentation :smile:

Reply:
I see, then this looks good to me :smile:

Reply:
I'm not sure how to solve this, @hdaljord . I've tried multiple ways of allowing both a boolean, boolish values in PropTypes - but the build keeps failing stating they are incompatible.
I tried various version of the one below, without any luck. 
```jsx
 focusable: PropTypes.oneOfType([
        PropTypes.bool,
        PropTypes.oneOf(["true", "false", "auto", "undefined"]),
    ])
```
When building I get the following error (in various variations):
```bash
  ../../node_modules/@types/react/index.d.ts:2381:9
    2381         focusable?: Booleanish | "auto";
                 ~~~~~~~~~
    The expected type comes from property 'focusable' which is declared here on type 'WeakValidationMap<IconProps>'
```
Essentially, just using ``bool`` will only allow true/false and not some of the other values (auto, undefined) which wasn't supported previously either. Not sure if we should bother fixing this?

Reply:
I assume you tried using
`
focusable: oneOfType([bool, string])
`
as used in Datepicker?

Wouldn't you get an error from propTypes if you pass in a string when the propType definition is bool? Or am I missing something here?

Reply:
I've tried to way you suggested previously, but it does unfortunately lead to the same error. A mismatch between TypeScript interface and PropTypes. 

One possible solution is to omit "focusable" from the SVGElement (and thus overriding it) and then add the enum check in PropTypes. This will as far as TypeScript is concerned allow any string, but PropTypes will do a enum check and throw a console.error if it's not in the allowed list. I've updated the code with this change. What do you think about this way of solving it? :thinking:

Reply:
I can't comment on the TS bit but I don't think we absolutely need a 1:1 relationship between TS and propTypes. TS users will still see the propType warnings if I'm not mistaken, so it's not a big issue if TS is looser than propTypes. If that makes sense.

Reply:
@steffenz, I think that is a better solution. And it even adds the possibility to pass in the valid values with propTypes. :+1: 

I think it's a good principle to have looser ts types and leave it to propTypes when it's not possible to make it the same. That way we don't give ts users more restrictions than js users.

Reply:
@ivarni That makes a lot of sense, and I agree. Thanks for your feedback. 
@hdaljord That's great to hear, and I agree with having looser ts types in such cases. Thanks for the feedback. 

If you think this is an acceptable solution, it would be great if you could mark it as approved :smile: